google/cloud/monitoring_v3/query.py

```diff
@@ -273,8 +273,14 @@ def select_metrics(self, *args, **kwargs):
 
             metric.label.<label> = "<value>"
 
-        However, by adding ``"_prefix"`` or ``"_suffix"`` to the keyword,
-        you can specify a partial match.
+        However, by adding ``"_notequal"`` to the keyword, you can inequality:
```

Comment:
Would you mind adding a link to https://cloud.google.com/monitoring/api/v3/filters#comparisons somewhere in this function's docstring?

Reply:
Sure thing. Done

Reply:
Thank you!